src/app/pages/account-order/account-order/account-order.component.html

```diff
@@ -63,7 +63,7 @@ <h1>{{ 'account.orderdetails.heading.default' | translate }}</h1>
   ></ish-line-item-list>
 
   <!-- Cost Summary -->
-  <div class="section cost-summary float-right">
+  <div class="section cost-summary">
```

Comment:
the summary widget should not be displayed with the full width, because labels and numbers are too far away from each other. That's why this float-right class cannot be removed

Reply:
looks broken to me xD